CONTRIBUTING.md

```diff
@@ -5,9 +5,9 @@ The search-gov repository includes the work of multiple contributors. You're enc
 #### Contributing to Search.gov Multilingual Search Support
 Note: As of November 2021, management of multilingual support has moved from the [GSA/Punchcard](https://github.com/GSA/punchcard/) repository to the [GSA/search-gov](https://github.com/GSA/search-gov) repository.
 
-The search-gov application uses the YAML files in [config/locales](https://github.com/GSA/search-gov/tree/master/config/locales) to provide localized translations of text strings based on the locale set for the user.
+The search-gov application uses the YAML files in [config/locales](config/locales) to provide localized translations of text strings based on the locale set for the user.
 
-You can edit existing files or create new files directly via the [Github web interface](https://github.com/GSA/search-gov/tree/master/config/locales). Or, you can use Git from the command line (see below). Either way, follow these steps to create a new localization file:
+You can edit existing files or create new files directly via the [Github web interface](config/locales). Or, you can use Git from the command line (see below). Either way, follow these steps to create a new localization file:
```

Comment:
I just realized that this link is incorrect. It needs to point to the main repo page (https://github.com/GSA/search-gov), NOT `config/locales` (because the web interface to add files is on the main page, but not sub-pages).

Reply:
![Screen Shot 2022-05-20 at 2 15 18 PM](https://user-images.githubusercontent.com/98416828/169611870-aa946641-b4ec-4b76-aa68-2107b3b543ad.png)

Maybe just return it to the original absolute link, https://github.com/GSA/search-gov/tree/master/config/locales ?

Reply:
@jamesmadhoc , apologies, I was either being blind or was looking at CONTRIBUTING.md (which doesn't have the "Add File" button because it's not a directory). So we can change that back to a relative link:
```
[Github web interface](config/locales)
```

Reply:
Took Config.md:10 back to the relative path, as above.